.github/workflows/pip_build_test.yaml

```diff
@@ -0,0 +1,43 @@
+# This workflow will install dependencies, install becquerel and run tests.
+# It runs on multiple python3 versions on macOS and Ubuntu.
+# It runs when a commit (merge) is push to the main (default) branch or for updates to a PR targeting main.
+# Based on: https://github.com/actions/starter-workflows/blob/main/ci/python-package.yml
+
+name: Python package
+
+on:
+
+  # When to test
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+
+jobs:
+  pip_build_test:
+
+    # OS and python combos to test
+    strategy:
+      matrix:
+        os: [ 'macos-latest', 'ubuntu-latest']
```

Comment:
windows?

Reply:
Good point. Here is the [OS listing](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources). Updated: 91df59000e5beb821aab244439abd1deb0b8ebcd

<img width="567" alt="image" src="https://user-images.githubusercontent.com/1504659/110147860-00146a80-7d91-11eb-8784-626fcc24643c.png">

Reply:
looks good. If it fails on windows, I would create a issue and take it off.